docs/alerts-and-notifications/creating-alerts-with-netdata-alerts-configuration-manager.md

```diff
@@ -1,44 +1,111 @@
 # Creating Alerts with Netdata Alerts Configuration Manager
 
-The Netdata Alerts Configuration Manager enables subscribers to easily set up Alerts directly from the Netdata Dashboard. More details on subscriptions can be found [here](https://www.netdata.cloud/pricing/).
+The **Netdata Alerts Configuration Manager** lets you create and fine-tune alerts directly from the Netdata Cloud Dashboard.  
+To use this feature, you’ll need an active Netdata subscription. → [View subscription plans](https://www.netdata.cloud/pricing/)
```

Comment:
This is a recurring thing, where we say "The **Netdata** this... and the **Netdata** that and you need the **Netdata** other"

cc @ilyam8 I feel we have also talked about this in the past, don't know what is best at the moment, because we name everything with the Netdata prefix by creation.

Reply:
Yes, after reading it, I felt that there was too much Netdata in this sentence, to the point that it was a parody on SEO optimization.

Reply:
Lets change to

> The **Netdata Alerts Configuration Manager by Netdata™** lets you create and fine-tune **Netdata** alerts directly from the **Netdata** Cloud Dashboard powered by **Netdata**'s proprietary **Netdata** Alert Technology™. To access this exclusive **Netdata** feature, you'll need an active Netdata subscription from **Netdata**, Inc. → [View **Netdata** subscription plans on Netdata.cloud](https://www.netdata.cloud/pricing/) and join the **Netdata** community of Netdata users using **Netdata**!